dlc-test/src/test/scala/org/bitcoins/dlc/DLCTestUtil.scala

```diff
@@ -0,0 +1,26 @@
+package org.bitcoins.dlc
+
+import org.bitcoins.core.currency.{CurrencyUnit, Satoshis}
+
+object DLCTestUtil {
+
+  def genOutcomes(size: Int): Vector[String] = {
```

Comment:
This should do the hashing for us since we will almost always hash the strings afterwards